patches/relay-mock-network-layer+1.2.0.patch

```diff
@@ -0,0 +1,124 @@
+patch-package
+deleted file mode 100644
+--- a/node_modules/relay-mock-network-layer/RelayMockNetworkLayerError.js
++++ /dev/null
+@@ -1,14 +0,0 @@
+-class RelayMockNetworkLayerError extends Error {
+-    constructor(errors) {
+-        const distinctMessages = errors
+-            .map(e => e.message)
+-            .filter((value, index, messages) => messages.indexOf(value) === index);
+-        super("RelayMockNetworkLayerError: " + distinctMessages.join(" - "));
+-        this.originalErrors = errors;
+-        if (Error.captureStackTrace) {
+-            Error.captureStackTrace(this, this.constructor);
+-        }
+-    }
+-}
+-
+-module.exports = RelayMockNetworkLayerError;
+\ No newline at end of file
+--- a/node_modules/relay-mock-network-layer/index.js
++++ b/node_modules/relay-mock-network-layer/index.js
+@@ -1,35 +1,43 @@
+-'use strict';
+-
+-const { makeExecutableSchema, addMockFunctionsToSchema } = require('graphql-tools');
+-const { graphql, printSchema, buildClientSchema } = require('graphql');
+-const RelayMockNetworkLayerError = require("./RelayMockNetworkLayerError");
+-
+-module.exports = function getNetworkLayer({schema, mocks, resolvers}) {
+-    return function fetchQuery(
+-        operation,
+-        variableValues
+-    ) {
+-        if (typeof schema === 'object' && schema.data) {
+-            schema = printSchema(buildClientSchema(schema.data));
+-        }
+-
+-        const executableSchema = makeExecutableSchema({typeDefs: schema, resolvers});
+-
+-        // Add mocks, modifies schema in place
+-        addMockFunctionsToSchema({ schema: executableSchema, mocks });
+-
+-        return graphql(executableSchema, operation.text, null, null, variableValues).then(
+-            // Trigger Relay error in case of GraphQL errors (or errors in mutation response)
+-            // See https://github.com/facebook/relay/issues/1816
+-
+-            result => {
+-                if (result.errors && result.errors.length > 0) {
+-                    return Promise.reject(new RelayMockNetworkLayerError(result.errors));
+-                }
+-                return Promise.resolve(result);
+-            }
+-        );
```

Comment:
This patch is changing this fix related to better reporting errors, so that’s probably not intentional?

Reply:
Just prettier reformatting, its all here: https://github.com/artsy/reaction/pull/1465/files/df0cd142d766c3abd6b0fb717008b6be0e9df6af#diff-d03959419fb73774b0d069d3d973b986R94

Reply:
ohh i see what you mean, at the top

Reply:
Yeah not sure where that top portion was even coming from? I popped off that file but if you add a log here from node modules it should work the same: https://github.com/1stdibs/relay-mock-network-layer/blob/master/index.js#L29

Reply:
Ahh, interesting: I know what this issue is: When `patch-package` ran it pulled in [your latest PR from upstream - 1.3.0](https://github.com/1stdibs/relay-mock-network-layer/pull/6) and performed a diff from there, vs diffing against the version currently installed in my local `node_modules` folder (1.2.1).